packages/react-ui/components/Autocomplete/__stories__/Autocomplete.stories.tsx

```diff
@@ -220,3 +384,31 @@ WithZeroWidth.story = {
     },
   },
 };
+// Выравнивание по правому краю
+export const WithRightTextAlignment: CSFStory<JSX.Element> = () => (
+  <UncontrolledAutocomplete
+    align={'right'}
+    source={['text', 'two', 'three']}
+  />
+);
+WithRightTextAlignment.story = {
+  parameters: {
+    creevey: {
+      tests: extendedTests('with right alignment', 'text'),
+    },
+  },
+};
+//Выравнивание по центру
```

Comment:
```suggestion

// Выравнивание по центру
```

Так уж исторически сложилось, что нужен пробел))
И ещё нужна пустая строка. С ней немного легче читать код.